### DISCUSSION TITLE
Search bar for CSS properties

### DISCUSSION THREAD

**Discussion by mo-lab** [webstudio-is/webstudio-community#5] [Ideas]
hi loved the product
I think a search bar for CSS attributes might be helpful
![Capture](https://user-images.githubusercontent.com/55297854/159508882-ae4a878d-63ef-4a12-9f49-875a697620e3.JPG)
right in this section

**Comment by kof** [comment]
Maybe even on top, inside the style panel, thought about it too, for the case where I know the property name, so I can just type right away

**Comment by kof** [comment]
Created an issue out of this original post, please follow it there, I will soon implement it https://github.com/webstudio-is/webstudio-designer/issues/15

**Reply by kof** [reply]
Was Implemented
